cloudsmith_cli/core/rest.py

```diff
@@ -12,7 +12,7 @@
 from cloudsmith_api.configuration import Configuration
 from cloudsmith_api.rest import ApiException, RESTClientObject
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry  # pylint: disable=import-error
+from urllib3.util.retry import Retry  # pylint: disable=import-error
```

Comment:
Just curious - why is pylint complaining about an import error here?

Reply:
Good catch. Changed the import but forgot to remove the pylint-disable. Removed it now.